compiler/rustc_mir_transform/src/coverage/spans/from_mir.rs

```diff
@@ -187,9 +187,7 @@ fn filtered_statement_span(statement: &Statement<'_>) -> Option<Span> {
         // for their parent `BasicBlock`.
         StatementKind::StorageLive(_)
         | StatementKind::StorageDead(_)
-        // Ignore `ConstEvalCounter`s
         | StatementKind::ConstEvalCounter
-        // Ignore `Nop`s
```

Comment:
I noticed that these comments were preventing `rustfmt` from formatting this match, presumably due to https://github.com/rust-lang/rustfmt/issues/4119.

Since these particular comments are low-value anyway, I worked around the issue by just deleting them.